SA3/qa_web/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from django.conf.urls import url
 from . import views
 
 urlpatterns = [
+    path(r'search/', include('haystack.urls')),
```

Comment:
This url is not present in any of the templates or views, therefore it is not reachable by regular means of navigation. A good place to fix this would be by adding a link in the navbar!

Reply:
Yes. And it's what i'm trying to do in next iteration: to add a input window on nav-bar. If it can't be finished, I will add a link in navbar.

Reply:
I think we should just solve this in this pull request. As the final Sprint deadline is closing in, it would be best to expedite things. Maybe just add a link in the navbar that links to `/search/`?